docs/rules/RemoveUnused.md

```diff
@@ -81,14 +84,14 @@ object Main {
   example match {
     case AB(_, _) => println("Not used")
   }
+  example match {
+    case AB(a, b) => println("Not used, but canonical names")
+  }
 }
 ```
 
-On Scala 3, `-Wunused:unsafe-warn-patvars` is required.
-
-On Scala 2.13.15+, canonical patterns (vars with the same names as the
-attributes) do not trigger unused warnings, so the input above will not
-be rewritten. See https://github.com/scala/bug/issues/13035.
```

Comment:
since this is now the default behavior on latest 2.13.x and 3.x, I am removing the "exception" note and documenting it via an example, as it can be surprising